editoast/migrations/2024-11-21-145205_create_stdcm_logs/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE stdcm_logs (
+    id int8 PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    trace_id text UNIQUE NOT NULL,
```

Comment:
```suggestion
    trace_id VARCHAR(X) UNIQUE NOT NULL,
```

where X is the maximum length of your trace_id

Reply:
Done.